### ISSUE TITLE
Feature request: emoji support

### ISSUE THREAD

**Issue by Offirmo** [yisibl/resvg-js#316]
Thanks for this amazing lib!

Unfortunately, it doesn't seem to be able to render emojis in text 😢

See this very simple SVG:
```svg
<svg xmlns='http://www.w3.org/2000/svg'
	viewBox='0 0 100 100'>
	<text y='.9em' font-size='90'>👨‍💻</text>
</svg>
```

It doesn't render anything visible.

When trying the playground https://resvg-js.vercel.app/ it doesn't render anything either.

Would it be feasible to add this? Thanks.

**Comment by yisibl** [comment]
I will be investigating this feature.

**Comment by qianshancun** [comment]
maybe there are copyright issues?

**Comment by Offirmo** [comment]
@qianshancun good point. The emojis would have to come from a font, in which case it would be up to the user to provide a color emoji font they have a license for.

We could rename this issue "support CBDT/CBLC color font format"  but I don't want to presume the best way to add this feature, I trust @yisibl ❤️

**Comment by yisibl** [comment]
resvg does not currently support emoji (aka color fonts), please follow: https://github.com/RazrFalcon/resvg/issues/487

**Comment by yisibl** [comment]
> maybe there are copyright issues?

Copyright depends on the specific font, beyond the scope of this question hair.

**Comment by Offirmo** [comment]
Thanks @yisibl for following up!

**Comment by are** [comment]
It seems that `resvg` merged https://github.com/RazrFalcon/resvg/issues/487 - are there any additional blockers of bringing it into `resvg-js`?

**Comment by yisibl** [comment]
> It seems that `resvg` merged [RazrFalcon/resvg#487](https://github.com/RazrFalcon/resvg/issues/487) - are there any additional blockers of bringing it into `resvg-js`?

This requires us to upgrade our resvg dependencies to the latest version. I don't have the time at the moment.

**Comment by Offirmo** [comment]
@yisibl it's completely ok, take your time! Thanks for this open source project!

**Comment by Offirmo** [comment]
@yisibl 🥺 🐩  pretty please 🙏

**Comment by facuparedes-litebox** [comment]
no updates yet? 😞

**Comment by lecramr** [comment]
@yisibl Is there any update on this?
